layouts/partials/header.html

```diff
@@ -0,0 +1,34 @@
+<!-- Mark up for Site Navigation Section-->
```

Comment:
Why do we need this for?

Reply:
I need to put the fail-script somewhere, and in the file that generates the menu seems the most logical. This  is copied from the theme.

We need changes in this file for the multi-lang stuff anyways, so not really a problem to include it now (rather than put the check somewhere illogical and then moving it when the multi-lang stuff comes around.

Also I prefer to go through merge-hell when I have to rebase the multi-lang stuff :)

Reply:
Best to read one commit at a time

Reply:
Even though it should be "Copy header from theme" not "from layouts". I'll fix this....